website/docs/using-unleash/compliance/fedramp.mdx

```diff
@@ -0,0 +1,54 @@
+|-----------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| [IA-02 Identification and Authentication](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=IA-2) (Organizational Users)                | Unleash provides single sign-on (SSO) to enable customers to enforce multi-factor authentication (MFA) for all Unleash users. |
+| [IA-02 (01) Identification and Authentication](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=IA-2) (Organizational Users); Multi-factor Authentication to Privileged Accounts     | Unleash provides SSO to enable customers to enforce multi-factor authentication (MFA) for all Unleash users.              |
+| [IA-02 (02) Identification and Authentication](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=IA-2) (Organizational Users); Multi-factor Authentication to Non-privileged Accounts | Unleash provides SSO to enable customers to enforce multi-factor authentication (MFA) for all Unleash users.              |
+| [IA-02 (08) Identification and Authentication](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=IA-2) (Organizational Users); Access to Accounts — Replay Resistant                  | Unleash restricts user logins after 10 failed attempts.                                                                   |
+
+## System and Communications Protection
+
+| **FedRAMP Control**                                                                                   | **Unleash Features**                                                                                                         |
+|-------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------|
+| [SC-08 (01) Transmission Confidentiality and Integrity](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=SC-8) (Cryptographic Protection) | Unleash implements cryptographic protection for data in transit, as detailed in our SOC2 report (available upon [request](mailto:sales@getunleash.io)). |
+| [SC-17 Public Key Infrastructure Certificates](https://csrc.nist.gov/projects/cprt/catalog#/cprt/framework/version/SP_800_53_5_1_0/home?element=SC-17)                              | Unleash uses PKI certificates issued by AWS and Google.                                                                    |
```

Comment:
Would service accounts as well as personal access tokens be based on PKI as well @dsusa72 ?

Reply:
I don't know, but I don't think it's relevant because we're not going to list here all the points in the solution where we use encryption. When we implement my suggestion in cell G15 (Encryption overview diagram), we will get down to this detail level.

Reply:
this section isn't about encryption, but rather PKI, e.g. certificates.  Saying that we use certificates only on AWS and Google, if we in fact use certificate-based auth on our API seems like an omission.  Unless of course we don't use certs on those services. Could an engineer validate this quickly?

Reply:
@michaelferranti , This was already validated with a principal eng. For clarification: we're not saying we use certificates "on" AWS & Google. We're saying "issued by" them, which is different. We're saying these are the only two CAs (Certification Authorities) that we get certificates from, as opposed to using certs from a Gov authority or self-signed ones.